myuw/templates/handlebars/card/profile/help_links_card.html

```diff
@@ -0,0 +1,20 @@
+{% load templatetag_handlebars %}
+{% tplhandlebars "profile_help_links_card" %}
+<div class="profile-links" style="position:relative;">
+  <h4>Related Help</h4>
+  <ul class="help_links employee_help_links">
```

Comment:
@mikeseibel Do we need "employee_help_links" here as a class? The list should serve both employees and students, right?